### PR TITLE
OSDOCS-1256, update Updating a cluster by using the web console procedure

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://issues.redhat.com/browse/OSDOCS-1256

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@rhamilto @spadgett PTAL

@kalexand-rh FYI on this change.

File: modules/update-upgrading-web.adoc [hunk 1]
```diff
@@ -47,9 +47,9 @@ such as `stable-{product-version}`.
 ====
 For production clusters, you must subscribe to a stable-* or fast-* channel.
 ====
-** If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
+** If the *Update Status* is not *Updates Available*, you cannot upgrade your
 cluster.
-** The *DESIRED VERSION* indicates the cluster version that your cluster is running
+** *Select Channel* indicates the cluster version that your cluster is running
```
Comment (rhamilto) [inline modules/update-upgrading-web.adoc:52]: `Current Version` or `Last 
```suggestion
** *Current Version* or *Last Completed Version* indicates the cluster version that your cluster is running
```

File: modules/update-upgrading-web.adoc [hunk 2]
```diff
@@ -60,15 +60,15 @@ ifdef::between[]
 the highest available version
 endif::between[]
 and click *Update*.
-The *UPDATE STATUS* changes to `Updating`, and you can review the progress of
+The *Update Status* changes to `Updating`, and you can review the progress of
```
Comment (rhamilto) [inline modules/update-upgrading-web.adoc:63]:
```suggestion
The *Update Status* changes to `Update to {version} in progress`, and you can review the progress of the cluster update by watching the progress bars for the operators and nodes.
```

File: modules/update-upgrading-web.adoc [hunk 3]
```diff
@@ -60,7 +60,7 @@ ifdef::between[]
 the highest available version
 endif::between[]
 and click *Update*.
-The *Update Status* changes to `Updating`, and you can review the progress of
+The *Update Status* changes to `Update to {version} in progress`, and you can review the progress of the cluster update by watching the progress bars for the operators and nodes.
 the Operator upgrades on the *Cluster Operators* tab.
```
Comment (rhamilto) [inline modules/update-upgrading-web.adoc:64]: Need to cut this line now.

**Review by kalexand-rh — COMMENTED** [review]
LGTM

**Comment by ahardin-rh** [comment]
@yapei Can you please QE review? Thank you!

**Comment by yapei** [comment]
@yanpzhan Can you please review this section?

File: modules/update-upgrading-web.adoc (hunk 1 above)
Comment (yanpzhan) [inline modules/update-upgrading-web.adoc:52]: Here should be:
”Last Completed Version“ indicates the cluster version that your cluster is running.

File: modules/update-upgrading-web.adoc [hunk 4]
```diff
@@ -47,9 +47,9 @@ such as `stable-{product-version}`.
...
 For production clusters, you must subscribe to a stable-* or fast-* channel.
 ====
-** If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
+** If the *Update Status* is not *Updates Available*, you cannot upgrade your
 cluster.
-** The *DESIRED VERSION* indicates the cluster version that your cluster is running
+** *Select Channel* indicates the cluster version that your cluster is running
 or is updating to.
 
 . Click *Updates Available*, select
```
Comment (yanpzhan) [inline modules/update-upgrading-web.adoc:55]: For 4.6, to choose a version to update, need click the button "Update". No button "Updates Available"

**Pull request merged**